lib/core/base/virtual-node/serial-virtual-node.js

```diff
@@ -44,14 +44,48 @@ class SerialVirtualNode extends AbstractVirtualNode {
   }
 }
 
+/**
+ * Map nodeName to nodeType
+ */
+const NodeNamesToTypesEnum = Object.freeze({
+  '#cdata-section': 2,
+  '#text': 3,
+  '#comment': 8,
+  '#document': 9,
+  '#document-fragment': 1
+});
```

Comment:
Is the comment necessary/sufficient? I'm also open to renaming, moving, or removing this.

Reply:
No, the comment isn't needed. I would say the `Objet.freeze` isn't need as well and you could remove that part of it (we only freeze publicly available things we don't want modified). Also we lowercase our consts (and you can drop the `enum`)

```js
nodeNamesToTypes
```